power_grid_model_c/power_grid_model/include/power_grid_model/common/common.hpp

```diff
@@ -91,4 +91,10 @@ using IntSVector = std::vector<IntS>;
 template <class T, class... Ts>
 concept is_in_list_c = (std::same_as<std::remove_const_t<T>, Ts> || ...);
 
+// functor to include all
+struct IncludeAll {
+    constexpr bool operator()(Idx /*ignored*/) const { return true; }
+};
+constexpr IncludeAll include_all{};
+
```

Comment:
is this the correct location for this? sounds too specific to be in the most root-level file. I'm OK with keeping it in `common/` but maybe in its own file?

If you prefer to keep it here, you can also just resolve my comment. I have no hard feelings

Reply:
I think it makes sense to be here. This functor gets used in different places (maybe even more in the future).

I can change it to more generic.

Reply:
Please have a look again. @mgovers